checker/check_test.go

```diff
@@ -27,8 +27,8 @@ func TestMakeCheckAnd(t *testing.T) {
 	t.Parallel()
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged